Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM scratch
+
+ENV PWD="/app"
+WORKDIR $PWD
+COPY slrp $PWD
+
+RUN printf '\
```

Comment:
Could you change it to env vars?

Every configuration property can be overridden through environment variable by using SLRP_ prefix followed by section name and key, divided by _. For example, in order to set log level to trace, do SLRP_LOG_LEVEL=TRACE slrp.

here is the example of env vars that would override defaults of the config file:
```
SLRP_APP_STATE=/var/db/slrp
SLRP_SERVER_ADDR=0.0.0.0:8089
SLRP_MITM_ADDR=0.0.0.0:8090
SLRP_HISTORY_LIMIT=1000
```

See the debugging hack that can simplify local dev: https://github.com/nfx/slrp/blob/main/main_test.go#L11-L18

Reply:
Done!